external-crates/move/crates/move-cli/src/base/new.rs

```diff
@@ -28,6 +29,11 @@ pub struct New {
     pub name: String,
 }
 
+fn is_valid_package_name(name: &str) -> bool {
+    let re = regex::Regex::new(r"^[a-z][a-z0-9_]*$").unwrap();
+    re.is_match(name)
```

Comment:
As mentioned in the earlier comments, use `IdentStr` to avoid duplicating the logic of what a valid package name is -- a valid package name is exactly a valid identifier.

```suggestion
    identifier::is_valid(name)
```

Where `identifier` is `move_core_types::identifier`.

Reply:
@amnn Got it! I use Identifier::is_valid to check `sui move new` package name validity. Thanks!